CHANGELOG.md

```diff
@@ -131,6 +131,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Tests with crud 1.4.0 (#336)
 - Tests with case sensitive SQL (#341)
 - Splice update operation accepts 3 arguments instead of 5 (#348)
+- Unable to use a slice of custom types as a slice of tuples or objects for
```

Comment:
It seems like a change that requires a migration (even though a simple one). So I think we should cover it in the corresponding README section.

Reply:
Thank you, updated.